src/components/ProjectDisciplines.js

```diff
@@ -65,12 +65,19 @@ export class ProjectDisciplines extends React.Component {
           text={'You must have an internet connection to use Zooniverse Mobile'} />
       </View>
 
+    const totalClassifications =
+      <StyledText
+        additionalStyles={[styles.totalClassifications]}
+        text={`${this.props.user.totalClassifications} total classifications`} />
```

Comment:
This is me being a grammar snob, but what happens when there is one total classification? Right now it's always total classifications (plural).